libraries/engage/filter/components/FilterPageContentWithProvider/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FilterPageProvider } from '../../providers';
+import Content from '../FilterPageContent';
+
+/**
+ * The FilterPageContent component
+ * @param {Object} props The component props
+ * @returns {JSX.Element}
+ */
+const FilterPageContent = ({
```

Comment:
the naming is a bit confusing between the two FilterPageContent components. Could we rename this one to include WithProvider?

Reply:
done